samples/name_domains_values/Coded_Domain_Reinforced.csv

```diff
@@ -0,0 +1,4 @@
+Name,Value
++,0
+-,1
++/-,2
```

Comment:
Those are the domains in the existing GDB, but feel free to improve when you create the new version.  I do think the Name field(vs. Value) needs to be "+" "-" "+/-" though since that is what we want to appear in the label in area F from Figure D-3/Table D-II.